### DISCUSSION TITLE
Is there a "manual" mode to trigger branch+PR only via Dependency Dashboard ?

### DISCUSSION THREAD

**Discussion by adriil** [renovatebot/renovate#20293] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.106.0

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

3.7.4

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

For the dependencies matching my packageRule, I would like Renovate to never create the branch and Pull Request by itself, but only on a manual trigger from Dependency Dashboard.
This is different from the `enabled: false` because we still want the vision of which dependencies can be updated.

### What you tried so far.

I tried to use `enabled: false` but it's not the good approach as it removes totally the vision of matching dependencies from the Dependency Dashboard.

**Comment by rarkins** [comment, marked as answer]
dependencyDashboardApproval=true

**Reply by adriil** [reply]
Thank you for putting my eyes in front of this obvious configuration setting :)

**Reply by HonkingGoose** [reply]
We call this the "Dependency Dashboard Approval workflow". Read more about it here:

https://docs.renovatebot.com/key-concepts/dashboard/#dependency-dashboard-approval-workflow
